go/nbs/table_manager.go

```diff
@@ -5,18 +5,18 @@
 package nbs
 
 type tableManager interface {
-	compact(mt *memTable, haver chunkReader) (name addr, chunkCount uint32)
-	open(name addr, chunkCount uint32) chunkSource
+	Compact(mt *memTable, haver chunkReader) (name addr, chunkCount uint32)
```

Comment:
the `tableManager` interface is starting to feel a little weird. it's fine for now, but it smells a bit bad that the `fileTableManager`s two methods just immediately call into either a static method or a specific implementation of another interface.